### PR TITLE
Isaac mir ver1

### PR DESCRIPTION
**Pull request by GueshSol**

### CODE REVIEW

**Comment by GueshSol** [comment]
I would be very happy if you provide me your comments (if any) for the betterment of this new work record. Thank you for your time.

File: new/LIT7150Miracle.xml [hunk 1]
```diff
@@ -0,0 +1,133 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7150Miracle" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+
+                <title xml:lang="gez" xml:id="t1">Miracle of St Mary on the monk Isacc who Prayed for St.Mary's Presence</title>
```
[the remark this answers is not in the document]
Comment (GueshSol) [reply new/LIT7150Miracle.xml:9]: Thank you @nafisa-valieva noted it:)

**Review by nafisa-valieva — COMMENTED** [review]
Thank you very much, Guesh! Do I understand correctly that this miracle was never edited or translated or discussed? I would imagine it was translated from Arabic and there must be something on it, no? 
Besides, if you have time, it would be very nice to have an incipit and explicit of the text.

**Comment by GueshSol** [comment]
Thank you @nafisa-valieva,  I will definitely add the incipits to both records. There is a significant amount of work to be done. I am currently writing an article that explores the conflation and blending of thematically related miracles during their transference through time. The first version, as it stands, is translated by Budge (1933) from MS [B-Oslo (SCOL) 248], folios 13r-14r. It seems that Budge may not have been aware of the two related versions. Additionally, there are three other miracles of this nature that exhibit similar thematic divergence.

**Comment by eu-genia** [comment]
please transfer this file to this branch #1249
one issue one branch
thank you

**Comment by eu-genia** [comment]
I have transferred the file, I will transfer the comments there as well and close here. please keep working in one branch. do not switch branches, do not create new records when they have been created, thank you.

**Pull request closed without merging**